content/developer/tutorials/discover_js_framework/02_build_a_dashboard.rst

```diff
@@ -322,17 +322,30 @@ of our users. In that case, it could make sense to lazy load our dashboard, and
 related assets, so we only pay the cost of loading the code when we actually want to
 look at it.
 
-To do that, we will need to create a new bundle containing all our dashboard assets,
-then use the `LazyComponent` (located in `@web/core/assets`).
+One way to do this is to use `LazyComponent` (from `@web/core/assets`) as an intermediate
+that will load an asset bundle before displaying our component.
+
+Example minimal usage of `LazyComponent`:
+
+.. code-block:: javascript
+
+   // example_action.js
+   export class ExampleComponentLoader extends Component {
+       static components = {LazyComponent};
+       static template = xml`
+           <LazyComponent bundle="'example_module.example_assets'" Component="'ExampleComponent'" />
+       `;
+   }
+
+   registry.category("actions").add("example_module.example_action", ExampleComponentLoader);
```

Comment:
When possible, use the `example` admonition for examples so you can tell them apart from the rest of the content.
```suggestion
.. example::

   .. code-block:: javascript

      export class ExampleComponentLoader extends Component {
          static components = {LazyComponent};
          static template = xml`
              <LazyComponent bundle="'example_module.example_assets'" Component="'ExampleComponent'" />
          `;
      }

      registry.category("actions").add("example_module.example_action", ExampleComponentLoader);
```